### PR TITLE
Use \overline rather than \bar in LaTeX particle names (CSV data files version 9)

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Closes https://github.com/scikit-hep/particle/issues/289.

(Funny enough I was already using overline when I introduced data on nuclei! At least now everything is fully consistent.)

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
OK, @henryiii, this is not ready. FYI @HDembinski.

**Comment by henryiii** [comment]
> this is **not** ready

Intentional, or did you mean "now"?

**Comment by eduardo-rodrigues** [comment]
> > this is **not** ready
> 
> Intentional, or did you mean "now"?

Apologies, meant "now ready" :D. I edited above.

File: src/particle/particle/utilities.py [hunk 1]
```diff
@@ -121,19 +121,47 @@ def str_with_unc(value, upper, lower=None):
...
+    \Lambda(1520)
+    >>> latex_name_unicode(n)
+    'Λ(1520)'
+    """
+    # Make sure "Lambda" and "lambda" are naturally deal with given that the
+    # unicodedata library uses "lamda" for "lambda" :S!
+    if "ambda" in name:
+        name = name.replace("ambda", "amda")
+    for gl in _list_name_greek_letters:
+        name = name.replace(r"\%s" % gl, "%s" % greek_letter_name_to_unicode(gl))
```
Comment (henryiii) [inline src/particle/particle/utilities.py:163]:
```suggestion
        name = name.replace(r"\{}".format(gl), greek_letter_name_to_unicode(gl))
```

**Review by henryiii — APPROVED** [review]
One minor cleanup (please use format instead of classic `%` formatting), then I'm happy.

**Comment by eduardo-rodrigues** [comment]
> One minor cleanup (please use format instead of classic `%` formatting), then I'm happy.

Thanks for spotting that. I still forget not to use "%" formatting, though I know that much better alternatives exist.

**Pull request merged**